src/routes/users.route.js

```diff
@@ -0,0 +1,64 @@
+import express from "express";
+import bcrypt from "bcryptjs";
+import { prisma } from "../lib/prisma.js";
+import { authenticate } from "../middlewares/auth.js";
+
+const router = express.Router();
+
+router.get("/me", authenticate, async (req, res) => {
+  const me = await prisma.user.findUnique({
+    where: { id: req.user.id },
+    select: {
+      id: true,
+      email: true,
+      nickname: true,
+      image: true,
+      createdAt: true,
+    },
+  });
+  res.json(me);
+});
+
+router.patch("/me", authenticate, async (req, res) => {
+  const { nickname, image } = req.body;
+  const updated = await prisma.user.update({
+    where: { id: req.user.id },
+    data: { nickname, image },
+    select: { id: true, email: true, nickname: true, image: true },
+  });
+  res.json(updated);
+});
+
+router.patch("/me/password", authenticate, async (req, res) => {
+  const { currentPassword, newPassword } = req.body;
+  const ok = await bcrypt.compare(currentPassword, req.user.password);
+  if (!ok)
+    return res
+      .status(400)
+      .json({ message: "현재 비밀번호가 일치하지 않습니다." });
+  const hashed = await bcrypt.hash(newPassword, 10);
+  await prisma.user.update({
+    where: { id: req.user.id },
+    data: { password: hashed },
+  });
+  res.json({ message: "비밀번호 변경 완료" });
+});
```

Comment:
```suggestion
router.patch("/me/password", authenticate, async (req, res) => {
  const { currentPassword, newPassword } = req.body;
  const ok = await bcrypt.compare(currentPassword, req.user.password);
  
  if (!ok)
    return res
      .status(400)
      .json({ message: "현재 비밀번호가 일치하지 않습니다." });
  
  const hashed = await bcrypt.hash(newPassword, 10);
  await prisma.user.update({
    where: { id: req.user.id },
    data: { password: hashed },
  });
  
  res.json({ message: "비밀번호 변경 완료" });
});
```
적절하게 공백을 주어서 가독성을 높여주면 좋을 것 같습니다.